### ISSUE TITLE
[Codex] Migrate WordPress.ORG API from Codex to DevHub

### ISSUE THREAD

**Issue by zzap** [WordPress/Documentation-Issue-Tracker#723]
- [ ] Migrate https://codex.wordpress.org/WordPress.org_API to DevHub (Common APIs)
- [ ] Review and update content where appropriate.

**Comment by zzap** [comment]
Heads up @WordPress/docs-issues-coordinators, we have a new issue open. Time to use 'em labels.

**Comment by TonyGravagno** [comment]
Before moving the page, there are a number of issues that could be addressed in the content.
https://wordpress.slack.com/archives/C02QB8GMM/p1696889892317979?thread_ts=1696876897.093749&cid=C02QB8GMM

Should I create individual tickets or can this comment serve as the bullet list?

**Comment by stevenlinx** [comment]
> Should I create individual tickets ...

I agree with what Beda mentioned in the Slack thread; creating a separate Issue would have been better.

The two tasks typically shall be in parallel:

- migrating the page as it is.
- adding/improving on top of the existing content (in the process, there could be new discussions on how to add/improve the content)

**Comment by TonyGravagno** [comment]
Created a new ticket #1255.
